scripts/build/cc/gcc.sh

```diff
@@ -280,6 +280,7 @@ do_gcc_core_backend() {
         gcc_build|gcc_host)
             CT_DoLog EXTRA "Configuring final gcc compiler"
             extra_config+=( "${CT_CC_SYSROOT_ARG[@]}" )
+            extra_config+=( "--with-headers=${CT_PREFIX_DIR}/${CT_TARGET}/include" )
```

Comment:
This piece doesn't seem motivated by the series description?

Reply:
I have updated the series/PR description.

Reply:
Thanks, this seems a lot more clear now. Happy to approve when you've switched it away from Draft.